internal_ws/ykcompile/src/arch/x86_64/mod.rs

```diff
@@ -333,9 +333,9 @@ fn local_to_reg_name(loc: &Location) -> &'static str {
 // Collection of functions required during a guard failure to instantiate and initialise the
 // stopgap interpreter.
 
-/// Given a vector of `FrameInfo`s `vptr`, instantiates and initialises a SIR interpreter and returns its
-/// pointer. Consumes `vptr`.
-extern "sysv64" fn invoke_sinterp(vptr: *mut Vec<FrameInfo>) -> *mut StopgapInterpreter {
+/// Given a pointer to a vector of `FrameInfo`s, creates and returns a boxed SIR interpreter.
```

Comment:
s/SIR interpreter/Stopgap interpreter

Reply:
Fixed in f1ec182ad469bce8ec189a2c012f6ba8c5be9ca2.